From: Caleb Connolly <caleb@connolly.tech>
Date: Sat, 8 May 2021 09:35:07 +0100
Subject: [PATCH 43/45] TEMP: dts: sdm845-oneplus: fix guard mem

The guard memory regions in this branch are wrong, the patch sent to
upstream is correct though, for now this commit will fix the version
here

Signed-off-by: Caleb Connolly <caleb@connolly.tech>
---
 arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi b/arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi
index 7774480..66bb4f3 100644
--- a/arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi
+++ b/arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi
@@ -51,24 +51,24 @@ reserved-memory {
 		 * it is otherwise possible for an allocation adjacent to the
 		 * rmtfs_mem region to trigger an XPU violation, causing a crash.
 		 */
-		rmtfs_lower_guard: memory@f5c00000 {
+		rmtfs_lower_guard: memory@f5b00000 {
 			no-map;
-			reg = <0 0xf5c00000 0 0x1000>;
+			reg = <0 0xf5b00000 0 0x1000>;
 		};
 		/*
 		 * The rmtfs memory region in downstream is 'dynamically allocated'
 		 * but given the same address every time. Hard code it as this address is
 		 * where the modem firmware expects it to be.
 		 */
-		rmtfs_mem: memory@f5c01000 {
+		rmtfs_mem: memory@f5b01000 {
 			compatible = "qcom,rmtfs-mem";
-			reg = <0 0xf5c01000 0 0x200000>;
+			reg = <0 0xf5b01000 0 0x200000>;
 			no-map;
 
 			qcom,client-id = <1>;
 			qcom,vmid = <15>;
 		};
-		rmtfs_upper: memory@f5e01000 {
+		rmtfs_upper_guard: memory@f5d01000 {
 			no-map;
 			reg = <0 0xf5d01000 0 0x2000>;
 		};
